FirstSolution/Part14Task2/Task.cs

```diff
@@ -0,0 +1,163 @@
+            Student student3 = new Student("Mankevich", "Ekaterina", "Andreevna", 2, new[] {6, 5, 7}, new[] {7, 7, 4},
+                new[] {8, 7, 6});
+            student1.PrintStudentInfo();
+            student2.PrintStudentInfo();
+            student3.PrintStudentInfo();
+            Console.WriteLine(student1.CountAverageGrade(student1.ProgrammingGrades));
+            Console.WriteLine(student1.CountAverageGrade(student1.AdministrationGrades));
+            Console.WriteLine(student1.CountAverageGrade(student1.DesingGrades));
+        }
+    }
+
+    public class Student
+    {
+        private string secondname;
+        private string firstname;
+        private string middlename;
+        private int group;
+        private int[] programmingGrades;
+        private int[] administrationGrades;
+        private int[] desingGrades;
+
+        public Student(string secondname, string firstname, string middlename, int group, int[] programmingGrades,
+            int[] administrationGrades, int[] desingGrades)
+        {
+            this.secondname = secondname;
+            this.firstname = firstname;
+            this.middlename = middlename;
+            this.group = group;
+            this.programmingGrades = programmingGrades;
+            this.administrationGrades = administrationGrades;
+            this.desingGrades = desingGrades;
+        }
+
+        public void PrintStudentInfo()
+        {
+            Console.WriteLine($"Students info : {firstname}, {middlename}, {secondname}, {group} group");
+            PrintStudentGrades();
+        }
+        
+        public void PrintStudentGrades()
+        {   
+            Console.WriteLine($"{firstname} {secondname}'s grades");
+            Console.WriteLine("Programming grades");
+            
+            for (int i = 0; i < programmingGrades.Length; i++)
+            {
+                Console.Write(programmingGrades[i] + " ");
+            }
+
+            Console.WriteLine("\nAdministration grades");
+            
+            for (int i = 0; i < administrationGrades.Length; i++)
+            {
+                Console.Write(administrationGrades[i] + " ");
+            }
+
+            Console.WriteLine("\nDesign grades");
+            
+            for (int i = 0; i < desingGrades.Length; i++)
+            {
+                Console.Write(desingGrades[i] + " ");
+            }
+            Console.WriteLine();
+        }
+
+        public float CountAverageGrade(int[] programmingGrades)
+        {
+            float sum = 0;
+            
+            for (int i = 0; i < programmingGrades.Length; i++)
+            {
+                sum += programmingGrades[i];
+            }
+
+            float averageScore = sum / programmingGrades.Length;
+            Console.WriteLine("Average programming grade");
+            return averageScore;
+        }
+        
+        public int GetProgrammingGrade()
+        {
+            return programmingGrades[2];
```

Comment:
Don't use specific index. Get and Set all Grades for particular array.

Reply:
Done